.github/workflows/security-audit-cron.yml

```
@@ -0,0 +1,20 @@
+name: Security audit cron job
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Would it make sense to combine the two files into one?
```suggestion
  schedule:
    - cron: "0 0 * * *"
  push:
    paths:
      - "**/Cargo.toml"
      - "**/Cargo.lock"
```

---

good idea, I just updated the PR